internal/controller/changetransferpolicy_controller.go

```diff
@@ -519,10 +711,32 @@ func (r *ChangeTransferPolicyReconciler) creatOrUpdatePullRequest(ctx context.Co
 			return fmt.Errorf("failed to create PR from branch %q to %q: %w", ctp.Spec.ProposedBranch, ctp.Spec.ActiveBranch, err)
 		}
 		r.Recorder.Event(ctp, "Normal", constants.PullRequestCreatedReason, fmt.Sprintf(constants.PullRequestCreatedMessage, pr.Name))
-		logger.V(4).Info("Created pull request")
+		logger.V(4).Info("Created pull request", "pullRequest", pr)
 		return nil
 	}
 
+	commitTrailers := trailers{}
+	commitTrailers[constants.TrailerPullRequestID] = pr.Status.ID
+	commitTrailers[constants.TrailerPullRequestSourceBranch] = pr.Spec.SourceBranch
+	commitTrailers[constants.TrailerPullRequestTargetBranch] = pr.Spec.TargetBranch
+	commitTrailers[constants.TrailerPullRequestCreationTime] = pr.Status.PRCreationTime.Format(time.RFC3339)
+	commitTrailers[constants.TrailerPullRequestUrl] = pr.Status.Url
+	pr.Spec.Commit.Message = fmt.Sprintf("%s\n\n%s\n\n%s", pr.Spec.Title, pr.Spec.Description, commitTrailers)
```

Comment:
Does this line do anything? Looks like setting the message happens in the retry function.